### DISCUSSION TITLE
Does AWC support using socket file ?

### DISCUSSION THREAD

**Discussion by loganmzz** [actix/actix-web#3131] [Q&A] [answered]
I'm consuming Podman API which by default is exposed as socket file. Before forcing using TCP, I'm wondering if I can exchange with Podman API using the default socket file.

Thank you for your help !

**Comment by robjtede** [comment, marked as answer]
`awc` doesn't have support for this yet.

I've opened this feature request issue: https://github.com/actix/actix-web/issues/3134

**Reply by loganmzz** [reply]
TY @robjtede !
